### PR TITLE
Feature/reporting functionality

### PR DESCRIPTION
**Pull request by BIPLAVGHOSAL**
- [x] add the function to count the scenarios from the path provided.
- [x] pass the mocha variable in run time, so that we can use that to pass values in the mochawesome Report.
- [x] updating Travis.yml so that build passes in CI.

### CODE REVIEW

File: dist/lib/yadda-core.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /* global featureFile, scenarios, steps */
 const Yadda = require('yadda');
 const wd = require('wd');
+var mocha = require('mocha');
```
Comment (ashishcentrica) [inline dist/lib/yadda-core.js]: const

File: package.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
 {
   "name": "ya-done-appium",
-  "version": "1.1.2",
+  "version": "1.1.3",
   "description": "Ready to use yadda - appium - chai framework",
   "repository": "https://github.com/britishgas-engineering/ya-done-appium",
-  "author": "Darren Corbett",
+  "author": "Biplav",
```
Comment (ashishcentrica) [inline package.json]: author: "Centrica" or ignore the field

**Review by ashishcentrica — APPROVED** [review]
minor suggestions, LGTM

File: package.json (hunk 2 above)
Comment (BIPLAVGHOSAL) [reply package.json]: I was thinking about what will be better as people will be moving here and there.

**Pull request merged**